host/src/server/api/v1/proof.rs

```diff
@@ -9,9 +8,12 @@ use crate::{
     interfaces::HostResult,
     metrics::{dec_current_req, inc_current_req, inc_guest_req_count, inc_host_req_count},
     proof::handle_proof,
+    server::api::v1::Status as StateResponse,
```

Comment:
```suggestion
    server::api::v1::Status,
```

Reply:
dmmm.. I misread the extract::State, thought name conflicts :(